doc/users/next_whats_new/2019-12-20-set_xy_position

```diff
@@ -0,0 +1,7 @@
+Add loc={'left'|'center'|'right'} to set_{}label()
```

Comment:
Proposal:
~~~
Align labels to axes edges
--------------------------
`~.axes.Axes.set_xlabel`, `~.axes.Axes.set_ylabel` and `ColorbarBase.set_label`
support a parameter ``loc`` for simplified positioning. Supported values are
'left', 'center', 'right' for x and and 'top', 'center', 'bottom' for y. The
default is controlled via :rc:`xaxis.labelposition` and
:rc:`yaxis.labelposition`; the Colorbar label takes the rcParam based on its
orientation.
~~~

- Curly braces are a bit hard to interpret here
- Link to the respective functions and rcParams
- I think we can leave out parameters x/y and horizontal alignment in this overview. They are probably rarely used currently and they are detailed in the `loc` docstring anyway.